src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ZeroFreeGlobalAllocUnicodeTests.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class ZeroFreeGlobalAllocUnicodeTests
+    {
+        [Fact]
+        public void ZeroFreeGlobalAllocUnicode_ValidPointer_Success()
+        {
+            IntPtr ptr = Marshal.StringToHGlobalUni("hello");
+            Marshal.ZeroFreeGlobalAllocUnicode(ptr);
```

Comment:
this is incorrect. memory allocated by Marshal.StringToHGlobalUni should be released by Marshal.FreeHGlobal

Reply:
similar to others. memory allocated by SecureStringToGlobalAllocUnicode should be released by Marshal.ZeroFreeGlobalAllocUnicode

Reply:
See https://github.com/dotnet/corefx/pull/31151/files